README.md

```diff
@@ -42,17 +42,70 @@ tutorial:
 
 ## What is Included
 
+This starter kit demonstrates communication with a Cardano Node using either
+the Node-to-Client or Node-to-Node operation modes of the Ouroboros network
+protocol and provides examples of multiple Ouroboros mini-protocols.
+
+- BlockFetch
+- ChainSync
+- LocalTxMonitor
+
+### BlockFetch
+
+The BlockFetch mini-protocol allows for fetching specific blocks from a remote
+Cardano Node using Node-to-Node communication over the network. The code for
+this example is in a single `main.go` file under `./cmd/block-fetch`.
+
+By default, it will fetch the first block of the Babbage Era on the Cardano
+mainnet from IOG's backbone servers. To change this, use the following
+environment variables:
+
+- `CARDANO_NODE_ADDRESS`: the address:port pair of a remote Cardano Node to
+  retrieve block
+- `CARDANO_NODE_HASH`: the block hash to fetch
+- `CARDANO_NODE_NETWORK`: named Cardano network to use to configure network
+  magic automatically
+- `CARDANO_NODE_NETWORK_MAGIC`: magic number used to identify a network
+- `CARDANO_NODE_RETURN_CBOR`: return raw CBOR bytes instead of describing text
+- `CARDANO_NODE_SLOT`: the slot in which the block hash was minted
```

Comment:
These should all be `BLOCK_FETCH_*`

Reply:
Nice catch. I wrote that before I switched it.